pkg/apis/core/v1alpha1/types_shoot.go

```diff
@@ -768,6 +768,8 @@ type Worker struct {
 	// CABundle is a certificate bundle which will be installed onto every machine of this worker pool.
 	// +optional
 	CABundle *string `json:"caBundle,omitempty"`
+	// Cri contains configurations of CRI support of every machine in the worker pool
+	Cri *Cri `json:"cri,omitempty"`
```

Comment:
```suggestion
	CRI *CRI `json:"cri,omitempty"`
```

also, change on the other places where used.